### PR TITLE
Add et7233 keyboard

### PR DESCRIPTION
**Pull request by yangzheng20003**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/et/et7233/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,218 @@
+{
+    "manufacturer": "HS",
+    "keyboard_name": "ET 7233",
+    "maintainer": "yangzheng20003",
+    "bootloader": "wb32-dfu",
+    "diode_direction": "ROW2COL",
+    "eeprom": {
+        "driver": "wear_leveling",
```
Comment (waffle87) [inline keyboards/et/et7233/keyboard.json:8]: Default behaviour.
```suggestion
```

File: keyboards/et/et7233/et7233.c [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    {0, CS18_SW9, CS17_SW9, CS16_SW9  },  // 55 F
+
+    {0, CS7_SW1,   CS8_SW1,   CS9_SW1    },  // 56 G
+    {0, CS7_SW2,   CS8_SW2,   CS9_SW2    },  // 57 H
+    {0, CS7_SW3,   CS8_SW3,   CS9_SW3    },  // 58 J
+    {0, CS7_SW7,   CS8_SW7,   CS9_SW7    },  // 59 K
+    {0, CS7_SW11,  CS8_SW11,  CS9_SW11   },  // 60 L
+    {0, CS7_SW12,  CS8_SW12,  CS9_SW12   },  // 61 ;:
+    {0, CS18_SW10, CS17_SW10, CS16_SW10  },  // 62 '
+    {0, CS18_SW11, CS17_SW11, CS16_SW11  }  // 63 #~
```
Comment (zvecr) [inline keyboards/et/et7233/et7233.c:84]: These are using the deprecated defines, the correct ones being in the format `SW1_CS1`.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/et/et7233/et7233.c (hunk 2 above)
Comment (yangzheng20003) [reply keyboards/et/et7233/et7233.c]: Thank you for your correction

File: keyboards/et/et7233/keyboard.json (hunk 1 above)
Comment (yangzheng20003) [reply keyboards/et/et7233/keyboard.json]: Thank you for reminding me, I have revised it.

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**